### PR TITLE
Who to teach subcategory page

### PR DESCRIPTION
**Pull request by emmaachesongray**
### Trello card
https://trello.com/c/kfhNP1d7/6528-phase-1-create-who-you-could-teach-sub-category

### CODE REVIEW

File: app/views/content/life-as-a-teacher/ages-and-specialisms/age-groups-you-could-teach.md [hunk 1]
```diff
@@ -1,12 +1,8 @@
 ---
-title: Who do you want to teach
-heading: Who do you want to teach?
-subcategory: Who to teach
+title: Age groups you could teach
+heading: Age groups you could teach
 description: |-
   Explore the key stages and ages you want to teach, whether that's primary, secondary, early years, further education, or pupils with special educational needs.
```
Comment (Sarah-DfE) [inline app/views/content/life-as-a-teacher/ages-and-specialisms/age-groups-you-could-teach.md:5]: Wonder if this should be 'Explore the key stages and ages you could teach/might want to teach'?

File: app/views/content/life-as-a-teacher/ages-and-specialisms/age-groups-you-could-teach.md [hunk 2]
```diff
@@ -62,7 +58,7 @@ Most disabled pupils and pupils with special educational needs learn in mainstre
 
 You can choose to apply for a teacher training course that specialises in SEND or you can gain the skills and experience as you train to teach.
 
-Learn more about the [routes to teach pupils with special educational needs and disabilities (SEND)](/life-as-a-teacher/teach-disabled-pupils-and-pupils-with-special-educational-needs).
+Learn more about the [routes to teach pupils with special educational needs and disabilities (SEND)](/life-as-a-teacher/ages-and-specialisms/special-educational-need).
```
Comment (Sarah-DfE) [inline app/views/content/life-as-a-teacher/ages-and-specialisms/age-groups-you-could-teach.md:61]: should the 's' go back on the end of 'need'?

File: app/views/content/life-as-a-teacher/teaching-as-a-career/career-progression.md [hunk 3]
```diff
@@ -71,7 +71,7 @@ Teacher training advisers Chris Farr, Fran Lee, Sian Mills and Elaine Boyd have
 
 ## What are teacher pastoral care roles?
 
-<strong>Fran:</strong>  Pastoral care in schools is looking at nurturing pathways. It’s about recognising all children have a right to feel safe and happy in school and guiding children on a more personal level. A [special educational needs co-ordinator (SENCO)](/life-as-a-teacher/teach-disabled-pupils-and-pupils-with-special-educational-needs) is part of pastoral care. Or you might be a form tutor and look after your pupils’ emotional wellbeing.
+<strong>Fran:</strong>  Pastoral care in schools is looking at nurturing pathways. It’s about recognising all children have a right to feel safe and happy in school and guiding children on a more personal level. A [special educational needs co-ordinator (SENCO)](/life-as-a-teacher/ages-and-specialisms/special-educational-need) is part of pastoral care. Or you might be a form tutor and look after your pupils’ emotional wellbeing.
```
Comment (Sarah-DfE) [inline app/views/content/life-as-a-teacher/teaching-as-a-career/career-progression.md:74]: as above with need

**Review by Sarah-DfE — COMMENTED** [review]
Some comments for you

**Review by gemmadallmandfe — COMMENTED** [review]
A few comments for you @emmaachesongray as requested!

File: app/views/content/life-as-a-teacher/age-groups-and-specialisms.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+---
+title: "Age groups and specialisms"
+heading: "Age groups and specialisms"
+category: Life as a teacher
+image: "static/images/content/hero-images/english.jpg"
+description: |-
+  Explore the different types of teacher you could be. From primary and secondary, to college and early years. You can also learn about how to train to specialise in teaching pupils with special education needs and disabilities (SEND).
```
[the remark this answers is not in the document]
Comment (emmaachesongray) [reply app/views/content/life-as-a-teacher/age-groups-and-specialisms.md:7]:
```suggestion
  Explore the different types of teacher you could be, from primary and secondary, to college and early years. You can also learn about teaching pupils with special education needs and disabilities (SEND).
```

File: app/views/content/life-as-a-teacher/ages-and-specialisms/special-educational-needs.md [hunk 5]
```diff
@@ -1,14 +1,10 @@
 ---
```
[the remark this answers is not in the document]
Comment (emmaachesongray) [reply app/views/content/life-as-a-teacher/ages-and-specialisms/special-educational-needs.md:1]: Yes, I removed it to shorten it. When I looked at semrush data, the keyword volumes were very low for teaching disabilities, whereas search terms around SEND or special educational needs were a lot higher. I'm not strongly against adding it in either though, so we can do that if you'd prefer?

[the remark this answers is not in the document]
Comment (gemmadallmandfe) [reply app/views/content/life-as-a-teacher/ages-and-specialisms/special-educational-needs.md:1]: I don't have a strong opinion - suggest discussing with Tom as to whether adding `disabilities` would help with re SEND terms or hinder with URL length?

**Review by martyn-w — DISMISSED** [review]
👍 LGTM

**Review by gemmadallmandfe — DISMISSED** [review]
Looks good to me

**Comment by gemmadallmandfe** [comment]
@emmaachesongray There is an issue with the breadcrumb - missing the Ages and specialisms page

image

image

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me

**Pull request merged**